### ISSUE TITLE
BlockLauncher Has Stopped

### ISSUE THREAD

**Issue by jmclean730** [zhuowei/MCPELauncher#1153]
My son is on a Samsung Galaxy Tab 4 8.0 (Android 5.1.1), and just installed BlockLauncher. He gets the error message "Unfortunately, BlockLauncher has stopped," when he tries to open the app. Is this device not supported, or is there a different issue?

Thanks for any help you can provide!

**Comment by cookmonster** [comment]
There updating

**Comment by cookmonster** [comment]
Sorry I know the feel I buy and now its updating

**Comment by lowjames** [comment]
Were working on it, sorry for the wait!

**Comment by lowjames** [comment]
Plz, close this report cause, were spending more time dealing with reports than porting BL!


Thx!

**Comment by MegaMix242** [comment]
I'm really disappointed in Bl. After mcpe 0.15.0 update it will send me a message Block launcher is stopped. Come on guys put the one star in Google play and tell to friends to do the same!!!!!

**Comment by lowjames** [comment]
Dont ,Were trying to deal with your issues instead of fixing Blocklauncher! Plz stop complaining!
Just waiiiiiiiit!

**Comment by jmclean730** [comment]
I appreciate the help and response...just wanted to make sure it wasn't the device my son is on.

Thx!

**Comment by lowjames** [comment]
It cold take up to 3 weeks to make the finalized update public, sorry for the wait!

**Comment by jmclean730** [comment]
Any progress? It's been the 3 weeks you noted, and my son asks daily. I'm
hoping to be able to give him good news soon.

Thanks!

**Comment by zedman2000** [comment]
@jmclean730 The betas have been coming out regularly.  Also, I'll point out with 0.15.3 released it has pushed this back yet again.  There are still a lot of issues with it at this time.  It'll show up on the play store when it's ready.  In the mean time give Toolbox by MrARM a try.  It has a subset of modpe working and has been updated to 0.15.3.  I've tried it with a few mods and they have all worked - I point out that the mods I've tried were smaller mods...so YMMV.  Sorry I don't have better news.  Follow Zhuowei on Twitter to get all the latest information on this.
